client/main.styl

```diff
@@ -0,0 +1,28 @@
+/* CSS declarations go here */
+
+.recentlyMentionedInfectiousAgentWord:hover{
+  cursor pointer
+  cursor hand
+  color blue
+}
+
+#spinner {
```

Comment:
Replace this with a class reference. I realize the ID reference is necessary in the JavaScript, so this may seem redundant, but we may add additional spinners. I generally like to avoid ID references in CSS.